src/sparseml/export/export.py

```diff
@@ -446,9 +452,30 @@ def main(
         integration=integration,
         sample_data=_parse_sample_data(sample_data),
         task=task,
+        **_parse_kwargs(kwargs),
     )
 
 
+def _parse_kwargs(kwargs: Optional[tuple] = None) -> Dict:
```

Comment:
would be good to have some sort of test for this and maybe extend as a util for other scripts